### PR TITLE
feat: full flexbox layout

### PR DESCRIPTION
**Pull request by hgwood**
Evenly divided width and 100% height at all zoom levels.

Full width, zoom 150%

image

Half-width, zoom 100%

image

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "graphql": "^0.7.0",
     "isomorphic-fetch": "^2.2.1",
     "react": "^15.2.1",
-    "react-ace": "^3.5.0",
+    "react-ace": "^4.0.0",
```
Comment (hgwood) [inline client/package.json:42]: Required to be able to override styles. See https://github.com/securingsincity/react-ace/commit/50e88e1417983c9349daefd2cd52fb44d9efc72c#diff-ac45432a2c51f02e94f43f5a1b4c8ef0

File: client/src/App.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class App extends Component {
   render () {
     return (
       <div>
-        <div style={{display: 'flex', flexDirection: 'row', height: '70em'}}>
+        <div style={{display: 'flex', flexDirection: 'row', height: '100vh'}}>
```
Comment (hgwood) [inline client/src/App.js:35]: Makes this `div` take up 100% of the screen estate, all the time. See `vh` at https://developer.mozilla.org/fr/docs/Web/CSS/length

File: client/src/FalcorPanel/index.js [hunk 3]
```diff
@@ -28,12 +28,10 @@ export default class FalcorPanel extends Component {
           mode="javascript"
           theme="github"
           name="falcor_editor"
-          fontSize={15}
-          width="400px"
-          height="680px"
           onChange={this.handleRequest}
           value={this.state.request}
           editorProps={{$blockScrolling: true}}
+          style={{flex: 1, width: null, height: null}}
```
Comment (hgwood) [inline client/src/FalcorPanel/index.js]: Forces `width` and `height` to be unset. If left undefined, they are set to `500px`

File: client/src/GraphQLPanel/graphiql.css [hunk 4]
```diff
@@ -18,10 +18,9 @@
     arial,
     sans-serif;
   font-size: 14px;
-  height: 680px;
   margin: 0;
   overflow: hidden;
-  width: 100%;
+  flex: 1;
```
Comment (hgwood) [inline client/src/GraphQLPanel/graphiql.css:23]: This is enough to make the editor take all the available space.

**Comment by hgwood** [comment]
Je vais m'occuper du conflit dans la journée.

**Comment by hgwood** [comment]
C'est good @antoinecellier

**Review by antoinecellier — APPROVED** [review]

**Pull request merged**